src/Components/Blogs/CurrentBlog.js

```diff
@@ -50,12 +50,12 @@ const CurrentBlog = (props) => {
     }
     setEditBlog(false);
     setBlogLoading(false);
-  }, []);
+  }, [id]);
```

Comment:
please dont remove these warnings due to useEffect they can cause some errors , so make every useeffect warning changes as it is.

Reply:
I got it